files/image_config/kubernetes/scripts/pod_hook.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
```

Comment:
This script is not used.

Reply:
This would be used by pod, declared through pod manifest.

Reply:
I see. I suggest you move this file to the future PR of pod manifest.

---
In reply to: [404436839](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r404436839) [](ancestors = 404436839)

Reply:
No. This **script is part of host image.** Manifest only refers it, as prestart/post-stop hook.

Reply:
File removed